### PR TITLE
Domain add remove endpoints are added

### PR DESCRIPTION
**Pull request by amertkara**
bumped to SDK version 1.42
updated CP deployment URL

JIRA ticket: https://appdirect.jira.com/browse/PI-12221

### CODE REVIEW

File: src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java [hunk 1]
```diff
@@ -41,4 +47,20 @@ public void triggerDomainOwnerhshipVerification(String accountId, String domain)
 						String.class
 				);
 	}
+
+	public void addDomain(String accountId, String domain) {
+		try {
+			restTemplate.postForLocation(format(DOMAINS_URL_TEMPLATE, chattyPieHost, accountId, domain), null);
```
Comment (hooriak) [inline src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java]: The chatty pie app already supports vanity domains?

File: src/main/java/com/chattypie/domain/ownership/verification/ChattyPieDomainOwnershipVerificationHandlerWithCallback.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 @Slf4j
 @RequiredArgsConstructor
-public class ChattyPieDomainOwnershiptVerificationHandlerWithCallback {
+public class ChattyPieDomainOwnershipVerificationHandlerWithCallback {
```
Comment (hooriak) [inline src/main/java/com/chattypie/domain/ownership/verification/ChattyPieDomainOwnershipVerificationHandlerWithCallback.java]: what was changed here?

File: src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java (hunk 1 above)
Comment (amertkara) [reply src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java]: Yes, it was added here
https://github.com/AppDirect/chatty-pie/pull/17

File: src/main/java/com/chattypie/domain/ownership/verification/ChattyPieDomainOwnershipVerificationHandlerWithCallback.java (hunk 2 above)
Comment (amertkara) [reply src/main/java/com/chattypie/domain/ownership/verification/ChattyPieDomainOwnershipVerificationHandlerWithCallback.java]: just fixed a typo `OwnershiptVerification`

File: src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java [hunk 3]
```diff
@@ -41,4 +47,20 @@ public void triggerDomainOwnerhshipVerification(String accountId, String domain)
...
+		} catch (DomainAlreadyExistsException e) {
+			log.error("Add domain request failed, response={}", e.getMessage());
```
Comment (tedturner29) [inline src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java]: can you add the exception to the end of the error log i.e. `log.error("Add domain request failed, response={}", e.getMessage(), e);`

Comment (tedturner29) [reply src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java]: can you also do this `response='{}'`? it helps splunk parse the log

Comment (amertkara) [reply src/main/java/com/chattypie/domain/ownership/verification/DomainOperationsService.java]: this is an expected exception, not sure if we should log the exception itself. The response should be enough. What do you think?

**Comment by tedturner29** [comment]
LGTM

**Comment by hooriak** [comment]
merging..

**Pull request merged**